packages/design-picker/src/components/unified-design-picker.tsx

```diff
@@ -0,0 +1,494 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+import { isEnabled } from '@automattic/calypso-config';
+import { MShotsImage } from '@automattic/onboarding';
+import { Button } from '@wordpress/components';
+import { useViewportMatch } from '@wordpress/compose';
+import { sprintf, hasTranslation } from '@wordpress/i18n';
+import { useI18n } from '@wordpress/react-i18n';
+import classnames from 'classnames';
+import { useTranslate } from 'i18n-calypso';
+import { noop } from 'lodash';
+import { useMemo } from 'react';
+import {
+	DEFAULT_VIEWPORT_WIDTH,
+	DEFAULT_VIEWPORT_HEIGHT,
+	MOBILE_VIEWPORT_WIDTH,
+} from '../constants';
+import {
+	getDesignPreviewUrl,
+	getMShotOptions,
+	isBlankCanvasDesign,
+	filterDesignsByCategory,
+	sortDesigns,
+} from '../utils';
+import BadgeContainer from './badge-container';
+import { UnifiedDesignPickerCategoryFilter } from './design-picker-category-filter/unified-design-picker-category-filter';
+import ThemePreview from './theme-preview';
+import type { Categorization } from '../hooks/use-categorization';
+import type { Design } from '../types';
+import './style.scss';
+
+const makeOptionId = ( { slug }: Design ): string => `design-picker__option-name__${ slug }`;
+
+interface DesignPreviewImageProps {
+	design: Design;
+	locale: string;
+	highRes: boolean;
+	verticalId?: string;
+}
+
+const DesignPreviewImage: React.FC< DesignPreviewImageProps > = ( {
+	design,
+	locale,
+	highRes,
+	verticalId,
+} ) => {
+	const scrollable = design.preview !== 'static';
```

Comment:
Now, we never return `preview` anymore from both static and generated designs, so `scrollable` will be always `true`. Can we simplify the logic then?